AppDevAnnouncements/Classes/NotificationView.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  NotificationView.swift
+//  AppDevAnnouncements
+//
+//  Created by Gonzalo Gonzalez on 10/16/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class NotificationView: UIView {
+
+    // Components
+    private let notificationContainerView = UIView()
+    private let dismissButton = UIButton()
+    private let visualImageView = UIImageView()
+    private let subjectLabel = UILabel()
+    private let bodyTextView = UITextView()
+    private let ctaButton = UIButton()
+
+    convenience init(dismissFunc: Selector, visualUrl: String, subject: String, body: String, ctaText: String, actionFunc: Selector, target: UIViewController) {
+        self.init()
+        dismissButton.addTarget(target, action: dismissFunc, for: .touchUpInside)
+        visualImageView.image = UIImage(named: visualUrl)
+        subjectLabel.text = subject
+        bodyTextView.text = body
+        ctaButton.setTitle(ctaText, for: .normal)
+        ctaButton.addTarget(target, action: actionFunc, for: .touchUpInside)
+    }
+
+    override init(frame: CGRect) {
```

Comment:
instead of having a `convenience init` and regular `init`, can we just just one init which takes in
```
init(
  dismissFunc: Selector, 
  visualUrl: String, subject: String, 
  body: String, ctaText: String, 
  actionFunc: Selector, 
  target: UIViewController
) {
  super.init(frame: .zero)
  ...
}
```